### PR TITLE
Issue210: Update / Improve comment for 'AccountCredentials'.

### PR DESCRIPTION
**Pull request by morrowc**
Update the docuemntation for AccountCredential to make it clear what the parts of the proto message mean, with respect to a normal openssh format ssh public key.

### CODE REVIEW

**Comment by morrowc** [comment]
I'd assign this to @limehat but apparently I can't do that?

**Comment by morrowc** [comment]
I'd also try to assign this to @melzhan but I can't do that either? 

how does review / assignment / review even work here?

**Comment by dplore** [comment]
> I'd also try to assign this to @melzhan but I can't do that either?
> 
> how does review / assignment / review even work here?

I created a team: https://github.com/orgs/openconfig/teams/gnsi-contributors/members
Gave it "triage" permissions (to manage PR's and Issues, but not write access).
I added @melzhan and @LimeHat to this contributors group.  I made @morrowc and admin, so you can add/remove people from here.

There is a maintainers team (https://github.com/orgs/openconfig/teams/gnsi-maintainers) which has write access.

**Review by melzhan — APPROVED** [review]

File: credentialz/credentialz.proto
Comment (LimeHat) [inline credentialz/credentialz.proto:441]: The format of this field (`authorized_key`) should be explicitly indicated, as Mel noted above, I think the above comment refers to the message instead

**Review by LimeHat — CHANGES_REQUESTED** [review]
If the non-zero key_type is mandatory, please indicate that in the comment.

In addition, if you want to document the message as a whole, note that the mapping between the enum `key_type` and the openssh key type is N:1 instead of 1:1 (for whatever reason we have three different RSA key types; all of which will be mapped to `ssh-rsa` in the openssh authorized_keys file format.

File: credentialz/credentialz.proto
Comment (morrowc) [reply credentialz/credentialz.proto]: done.

File: credentialz/credentialz.proto
Comment (morrowc) [reply credentialz/credentialz.proto:443]: done, moved the comment up for mel's comment and clarified that the key-type is mandatory.

**Comment by melzhan** [comment]
It looks like I can't approve this and it needs approval from Rob or Marcus

**Pull request merged**
